packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -86,13 +88,28 @@ const ToggleMenuDrawer = React.forwardRef(
         const [secondary_routes_config, setSecondaryRoutesConfig] = React.useState([]);
         const [is_submenu_expanded, expandSubMenu] = React.useState(false);
 
-        const { is_dashboard } = React.useContext(PlatformContext);
+        const { is_deriv_crypto } = React.useContext(PlatformContext);
```

Comment:
I think we have removed `is_deriv_crypto` from `PlatformContext`

Reply:
Thanks, I removed this now 👍🏻